src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -160,7 +165,9 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "lark-parser": ("lark",),
     "launchdarkly-server-sdk": ("ldclient",),
     "mail-parser": ("mailparser",),
+    "matplotlib": ("matplotlib", "mpl_toolkits"),
```

Comment:
```suggestion
    "matplotlib": ("matplotlib", "mpl_toolkits",),
```

Reply:
weird that i see lines with: `"setuptools": ("easy_install", "pkg_resources", "setuptools"),`